solutions/src/main/java/ru/sberbank/school/task02/ServiceFactoryImpl.java

```diff
@@ -0,0 +1,20 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.exception.FxConversionException;
+
+public class ServiceFactoryImpl implements ServiceFactory {
+
+
+
+    @Override
+    public FxConversionService getFxConversionService(ExternalQuotesService externalQuotesService) throws FxConversionException {
+
+        if(externalQuotesService == null)
```

Comment:
&#39;if&#39; construct must use &#39;{}&#39;s.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1328591503>*